### DISCUSSION TITLE
ThingsOn MQTT Bench

### DISCUSSION THREAD

**Discussion by volkanalkilic** [dotnet/MQTTnet#1675] [Show and tell]
I just released ThingsOn MQTT Bench, a simple console application to benchmark MQTT brokers. You can use it to test the performance of popular MQTT brokers like Mosquitto, HiveMQ, EMQX, VerneMQ, and ActiveMQ.

https://github.com/volkanalkilic/ThingsOn.MQTT.Bench

**Comment by Int32Overflow** [comment]
Do you have results from the benchmark that compares MqttNet with other Mqtt brokers? Can you publish the results?
